### DISCUSSION TITLE
开启”实验性：绕过中国大陆 IP“后adguardhome大陆部分广告屏蔽失效

### DISCUSSION THREAD

**Discussion by zzz6839** [vernesong/OpenClash#2768] [Q&A] [answered]
adguardhome配置：
![image](https://user-images.githubusercontent.com/57741549/197925132-cd435bee-888d-4efa-9f4a-96f13d5b5855.png)
![image](https://user-images.githubusercontent.com/57741549/197925215-e34915b8-343c-446f-9946-be860a1a05db.png)
openclash配置：
![image](https://user-images.githubusercontent.com/57741549/197925306-a2fb9461-8aab-46bc-9b27-3978444b95b8.png)

想请教下大佬们该如何处理。

**Comment by zheshifandian** [comment]
重启路由器一般就解决了
可以用这个网站测一下
https://d3ward.github.io/toolz/adblock

**Reply by zzz6839** [reply]
感谢回复，重启后并没有解决国内广告屏蔽失效的问题，并且dashboard上还有少量CHINAIP残留，估计我们的设定有出入。
![image](https://user-images.githubusercontent.com/57741549/197983977-46fdee86-f703-446b-939b-df0e627fbc86.png)

**Comment by Karmylr** [comment]
大陆域名不经过clash，自然无法让dns请求走adguard home了

**Comment by zzz6839** [comment, marked as answer]
通过docker装了个adguardhome然后openclash里大陆域名 DNS 服务器设置为docker的解决，现在软路由上两个adguardhome一个主外一个主内，英雄联盟，爱来自瓷器。

**Reply by dayuanpan** [reply]
大佬能说下具体怎么实现2个adg分别主内和主外的吗？
